.github/workflows/releaser.yml

```diff
@@ -0,0 +1,26 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
If I read that correctly this will create a release on every tag, wouldn't it be better to only release on `v.*`?

Reply:
That is a good point

Reply:
Will we have any tags that are not v[semver]? We need that format for Go and our internal library. The '*' may work for now.

Reply:
I guess that its a safe option in case we mess up with the tags? I dont mind adding that.

Reply:
Well we could have hotfixes that are tagged and do not have an automated release...

Reply:
updated to add the `v.*` filter to be safe!